diff --git org/xwiki/wiki/template/script/WikiTemplateManagerScript.java org/xwiki/wiki/template/script/WikiTemplateManagerScript.java
index 74ca25f..8e49bf9 100644
--- org/xwiki/wiki/template/script/WikiTemplateManagerScript.java
+++ org/xwiki/wiki/template/script/WikiTemplateManagerScript.java
@@ -40,6 +40,7 @@
 import org.xwiki.wiki.descriptor.WikiDescriptor;
 import org.xwiki.wiki.descriptor.WikiDescriptorManager;
 import org.xwiki.wiki.manager.WikiManagerException;
+import org.xwiki.wiki.provisioning.WikiProvisioningJob;
 import org.xwiki.wiki.template.WikiTemplateManager;
 import org.xwiki.wiki.template.WikiTemplateManagerException;
 
@@ -223,7 +224,11 @@ public Exception getLastException()
     public JobStatus getWikiProvisioningJobStatus(List<String> jobId)
     {
         try {
-            return wikiTemplateManager.getWikiProvisioningJob(jobId).getStatus();
+            WikiProvisioningJob wikiProvisioningJob = wikiTemplateManager.getWikiProvisioningJob(jobId);
+            if (wikiProvisioningJob == null) {
+                return null;
+            }
+            return wikiProvisioningJob .getStatus();
         } catch (WikiTemplateManagerException e) {
             return null;
         }
